.idea/compiler.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
You just did the mistake that all git newcomers are doing: committing to the repository autogenerated files and editors configuration. They shouldn't be part of the repository and should stay locally. So, please, remove the `.idea` and `target` directories from the git. Be careful, you shouldn't remove them at all, but only from the git. As far I remember, `git rm -r --cached .idea target` should do the trick.

Next step is to adding these directories to `.gitignore` file to tell the git to ignore them (when next time you will use `git add .` for example). See here for details: https://help.github.com/articles/ignoring-files/